src/multiParamManager.js

```diff
@@ -2,6 +2,7 @@
 
 var yo = require('yo-yo')
 var css = require('./universal-dapp-styles')
+// var helper = require('../remix/remix-lib/src/execution/txHelper')
```

Comment:
we should perhaps remove the comment if this is not going to be readded in the future